packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/nodejs/integrationsteps.md

```diff
@@ -51,31 +56,63 @@ If the org's application is properly configured with multiple factors, `idx.regi
+ If the call is successful, the method returns a status of `Idx.Status:PENDING` and a `nextStep` field requiring verification, which indicates that the SDK is ready for the email verification code. The next step is to redirect the user to the email verification code page, as shown in the SDK sample application route to `/enroll-authenticator`.
+
+<div class="common-image-format">
+
+![Displays an Enroll email authenticator page screenshot that includes field for the email verification code. A Verify button continues the flow.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-self-serv-screen-email-verify-nodejs.png)
+
+</div>
 
 ### Step 6: User submits email verification code
 
 The next step is to call `idx.register` again passing in the verification code. In the email verification use case, the code passed into the method is the code found in the verification email.
 
 Based on the configuration described in [Set up your Okta org for multifactor use cases](/docs/guides/oie-embedded-common-org-setup/java/main/#set-up-your-okta-org-for-multifactor-use-cases), the app in this use case is set up to require one possession factor (either email or phone). After the email factor is verified, the phone factor becomes optional. In this step, the `nextStep` field can include `canSkip` for the phone authenticator. You can build a **Skip** button in your form to allow the user to skip the optional phone factor.
 
+<div class="common-image-format">
+
+![Displays a Select authenticator page screenshot that includes a selection for Phone authentication. A Select button continues the flow. A Skip button skips the phone authentication flow. ](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-self-serv-screen-phone-auth-and-skip-nodejs.png)
+
+</div>
+
 If the user decides to skip the optional factor, they are considered signed in since they have already verified the required factors. See [Step 8, Option 1: Skip phone factor](#step-8-option-1-skip-phone-factor) for the skip authenticator flow. If the user decides to select the optional factor, see [Step 8, Option 2: User selects phone authenticator](#step-8-option-2-user-selects-phone-authenticator) for the optional phone authenticator flow.
 
 ### Step 7, Option 1: Skip phone factor
 
-If the user decides to skip the phone factor enrollment, make a call to `idx.register` passing in the value `{skip: true}`. This method skips the authenticator enrollment, as shown on the `Authenticator.js` from the SDK sample application.
+If the user decides to skip the phone factor enrollment, make a call to `idx.register` passing in the value `{skip: true}`. This method skips the authenticator enrollment, as shown in `Authenticator.js` from the SDK sample application.
```

Comment:
```suggestion
If the user decides to skip the phone factor enrollment, make a call to `idx.register` that passes the value `{skip: true}`. This method skips the authenticator enrollment, as shown in `Authenticator.js` in the SDK sample application.
```

Reply:
I'm going to not commit this suggestion, although the passage could be better worded. I.e calling the method doesn't pass the value; you need to call the method with the value (passed in).